src/App.js

```diff
@@ -1,45 +1,97 @@
 import './styles/reset.css'
 import './styles/index.css'
-
 import initialStoreItems from './store-items'
+import StoreItem from './components/StoreItem'
+import CartItem from './components/CartItem'
+import { useState } from 'react'
+import { Total } from './components/Total'
 
-/*
-Here's what a store item should look like
-{
-  id: '001-beetroot',
-  name: 'beetroot',
-  price: 0.35
-}
-
-What should a cart item look like? 🤔
-*/
+export default function App() {
+  const [storeItems, setStoreItem] = useState(initialStoreItems)
+  const [cartItems, setCartItems] = useState([])
 
-console.log(initialStoreItems)
+  function AddToCart(storeItem) {
+    //if item already in the cart
+    const found = cartItems.find(item => item.id === storeItem.id)
+    if (found) {
```

Comment:
I love the use of putting your find statement in a const then using it in your if statement, makes the code more readable